### PR TITLE
feat: making tasklist title dynamic uppon parameters

### PR DESCRIPTION
**Pull request by vsgoulart**
## Related issues

closes https://github.com/camunda/tasklist/issues/4758

[Test env](https://hel-1.tasklist.ultrawombat.com/e5fec69b-1b47-4c6e-938d-36f80f2e4f83/new/subscribeFormProcess)

### CODE REVIEW

**Review by douglasbouttell-camunda — APPROVED** [review]

File: tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/api/rest/v1/controllers/NewController.java [hunk 1]
```diff
@@ -33,11 +36,35 @@ public class NewController {
 
   @Autowired private ServletContext context;
```
Comment (nathansandi) [inline tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/api/rest/v1/controllers/NewController.java:37]: You can delete this line, it looks it is not on usage.

File: tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/api/rest/v1/controllers/NewController.java [hunk 2]
```diff
@@ -33,11 +36,35 @@ public class NewController {
```
Comment (nathansandi) [inline tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/api/rest/v1/controllers/NewController.java:36]: Is there any reason to call this class "NewController"?

Would be possible rename by the domain like `PublicProcessController` ? 

What do you think?

**Comment by nathansandi** [comment]
Thanks for work on BE implementation as well, Great work
Just some small points in addition to my comments

- Would be possible add the unit tests over the controller layer?
- Do you have a path in case process is not found?

**Comment by vsgoulart** [comment]
@nathansandi should be ready for review again

**Comment by vsgoulart** [comment]
> Thanks for work on BE implementation as well, Great work Just some small points in addition to my comments
> 
>     * Would be possible add the unit tests over the controller layer?
> 
>     * Do you have a path in case process is not found?

Missed this comment

**Comment by marcosgvieira** [comment]
@nathansandi I added tests for the PublicController, please have a look.

**Review by nathansandi — APPROVED** [review]
LGMT, thanks for commit the changes.

**Pull request merged**